### PR TITLE
Typos on "numbers" concept.

### PR DESCRIPTION
**Pull request by rcardenes**
Closes [#2490](https://github.com/exercism/python/issues/2490)
Found a few things to fix in Currency Exchange and Meltdown Mitigation

### CODE REVIEW

File: exercises/concept/currency-exchange/.docs/hints.md [hunk 1]
```diff
@@ -17,13 +17,13 @@
 ## 4. Calculate number of bills
 
 - You need to divide `budget` into `denomination`.
-- You need to use type casting _int_ to get exact number of bill.
+- You need to use type casting _int_ to get exact number of bills.
```
Comment (BethanyG) [inline exercises/concept/currency-exchange/.docs/hints.md:20]:
```suggestion
- You need to use type casting _int_ to get the exact number of bills.
```

File: exercises/concept/currency-exchange/.docs/hints.md [hunk 2]
```diff
@@ -17,13 +17,13 @@
...
 - To remove decimal places from a `float`, you can convert it to `int`.
 
   **Note:** The `//` operator also does floor division. But, if the operand has `float`, the result is still `float`.
 
 ## 5. Calculate exchangeable value
 
 - You need to calculate `spread` percent of `exchange_rate` using multiplication operator and add it to `exchange_rate` to get the exchanged currency.
-- Actual rate need to be computed! add exchange rate and exchange fee.
+- Actual rate needs to be computed! Add exchange rate and exchange fee.
```
Comment (BethanyG) [inline exercises/concept/currency-exchange/.docs/hints.md:28]:
```suggestion
- The actual rate needs to be computed.  Remember to add exchange rate and exchange fee.
```

File: exercises/concept/meltdown-mitigation/.docs/hints.md [hunk 3]
```diff
@@ -10,7 +10,7 @@
 
 - Comparison operators and boolean operations can be combined and used with conditionals.
 - Conditional expressions must evaluate to `True` or `False`.
-- `else` can be used for a code block that will execute when a conditional test returns `False`.
+- `else` can be used for a code block that will execute when all conditional tests returns `False`.
```
Comment (BethanyG) [inline exercises/concept/meltdown-mitigation/.docs/hints.md:13]:
```suggestion
- `else` can be used for a code block that will execute when all conditional tests return `False`.
```

File: exercises/concept/meltdown-mitigation/.docs/instructions.md [hunk 4]
```diff
@@ -5,18 +5,18 @@ In this exercise, we'll develop a simple control system for a nuclear reactor.
...
 If the reactor state goes beyond criticality, it can overload and result in a meltdown.
-We want to mitigte the chances of meltdown and correctly manage reactor state.
+We want to mitigate the chances of meltdown and correctly manage reactor state.
 
 The following three tasks are all related to writing code for maintaining ideal reactor state.
 
 ## 1. Check for criticality
 
-The first thing a control system has to do is check if the reactor is balanced in criticality.
+The first thing a control system has to do is checking if the reactor is balanced in criticality.
```
Comment (BethanyG) [inline exercises/concept/meltdown-mitigation/.docs/instructions.md:14]:
```suggestion
The first thing a control system has to do is check if the reactor is balanced in criticality.
```

Comment (BethanyG) [reply exercises/concept/meltdown-mitigation/.docs/instructions.md:14]: This phrasing (as strange as it sounds) is an imperative (_has to do a check_ ) but maybe the whole thing should be rephrased into _`"Checking if the reactor is balanced in criticality is the first thing a control system has to do."`_  for clarity?

**Review by BethanyG — COMMENTED** [review]
@rcardenes - Thank you so much for this!  I really appreciate the thorough review.  I had a couple of quibbles/suggestions, but this otherwise looks excellent.  🌟

**Comment by rcardenes** [comment]
Yeah .. Sorry for taking so long. When I was about to get to it I needed to spend a couple of days off at home.

**Review by BethanyG — APPROVED** [review]
Approving and merging! 🌟

**Pull request merged**